src/shared/clients/lambdaClient.ts

```diff
@@ -11,8 +11,15 @@ import { getLogger } from '../logger'
 import { ClassToInterfaceType } from '../utilities/tsUtils'
 
 export type LambdaClient = ClassToInterfaceType<DefaultLambdaClient>
+
+const fiveMinutesInMillis = 5 * 60 * 1000
```

Comment:
I would probably name this to defaultTimeout or defaultLambdaTimeout, since we can technically have this be any timeout number in the future

Reply:
I changed it a bit how I'm declaring this. I'll put a comment to tell what is the amount.